spyder/widgets/editor.py

```diff
@@ -1005,16 +1005,8 @@ def get_tab_tip(self, filename, is_modified=None, is_readonly=None):
             else:
                 return text % (osp.basename(filename), osp.dirname(filename))
 
-    def __get_sorting_func(self):

```

Comment:
Is this a method we would like to preserve in case the user want to sort (by adding an option in the editor menu?)

@ccordoba12 ?